panel/src/panel/upload.js

```diff
@@ -244,6 +244,10 @@ export default (panel) => {
 					filename: file.name + "." + file.extension,
 					url: this.url,
 					error: (xhr, src, response) => {
+						if (panel.debug) {
```

Comment:
Why don't we use panel.error here as well?

Reply:
Cause I'm stupid 😆

Reply:
That's what you said 🤣